catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -37,6 +41,10 @@
 
     private static final Color SUCCESS_COLOR = Ansi.Color.GREEN;
 
+    private static final Color INFO_COLOR = Ansi.Color.BLUE;
```

Comment:
❓ Looks like we're using `CYAN` in other commands for information messages. Couldn't find `BLUE` in any of our commands. Too bad we still don't have common constants for these...